### DISCUSSION TITLE
connectToRemoteAndInitialize isn't defined

### DISCUSSION THREAD

**Discussion by jstride** [simolus3/drift#2240] [Q&A] [answered]
I'm trying out drift for a project and following the example app. I get the error that connectToRemoteAndInitialize is defined in the web.dart file. What am I missing? https://github.com/simolus3/drift/blob/18c6139eb35c572be8effadc24f7f9437ecb0b39/examples/app/lib/database/connection/web.dart

**Comment by simolus3** [comment, marked as answer]
Ah sorry - the example in this repository uses the latest API on the same branch, which in that case hadn't been released yet.

`connectToRemoteAndInitialize` is available in drift version 2.4.0, which is available on pub.dev now.

**Comment by jstride** [comment]
Thanks @simolus3 that works perfectly now
